endpoint/version/endpoint.go

```diff
@@ -29,16 +29,6 @@ type Config struct {
 	Service *version.Service
 }
 
-// DefaultConfig provides a default configuration to create a new version
-// endpoint by best effort.
-func DefaultConfig() Config {
-	return Config{
-		// Dependencies.
-		Logger:  nil,
-		Service: nil,
-	}
-}
```

Comment:
It would be perfect to also split `DefaultConfig` removal. But it's not a blocker from my side.

Reply:
Moved this here https://github.com/giantswarm/microendpoint/pull/30